terraform/modules/front_door/waf.tf

```diff
@@ -152,4 +157,97 @@ resource "aws_wafv2_web_acl" "this" {
       sampled_requests_enabled   = true
     }
   }
+
+  rule {
+    name     = "login-ip-rate-limit"
+    priority = 7
+
+    action {
+      block {
+        custom_response {
+          response_code = 429

```

Comment:
I believe there is application logic that ties a 429 HTTP response to a "too many requests" error page, so thought it might be good to induce this on the two rate limiting rules we have here

Reply:
Having investigated further, it doesn't look like the application automatically redirects to this page if it sees a 429 response. Seems to be only if we're using rack attack for rate limiting, do you know if this is expected? Nonetheless I think it is worth still responding with a 429 (unless we don't obfuscate there is rate limiting from the client?)

Reply:
So I think when rack attack rate limits (which we may only have set up on password reset requests?) it actually forwards to a "/429" route and we map that to an error page. 

If we return a 429 here it'll be before it hits the application, so I'm not expecting we can do nice in-app errors. We presumably could set this up to return a nicer thing, but not bothered about it at the moment - think just responding with a 429 will do.